ISSUE_TEMPLATE

```diff
@@ -0,0 +1,20 @@
+#### Steps to reproduce or link to a repro project:
+
+
+#### Expected  behavior:
+
+
+#### Actual behavior:
+
+
+#### Version of `Microsoft.AspNetCore.App` or `Microsoft.AspNetCore.All` used:
```

Comment:
Might want more details e.g. tfm since desktop does not reference these packages.